conf/receiver.cfg

```diff
@@ -25,17 +26,22 @@ check_crls: false
 
 [messaging]
 # If using AMS this is the project that SSM will connect to. Ignored for STOMP.
-ams_project:
+ams_project: accounting
 
 # Destination to which SSM will listen.
-destination: /queue/ssm2test
+destination: ssm2test
```

Comment:
If ssm2test is a dedicated testing architecture, then this is fine, but the intent is for this to be deliberately changed by the user.  So, 'changeme' or blank would be good.

Reply:
Done, though went with the blank option.